README.md

```diff
@@ -2,53 +2,199 @@
 
 # NetKit.Chat
 
-Open source product is used to communicate people in real time. 
-This project uses ASP.NET Core 2.0
+NetKit.Chat is open source product which used to communicate people in real time. Chat provide you ability to send group and direct messages including images, manage groups and receive push notifications.
+This project uses ASP.NET Core 2.2 version.
 
-## Development setup
+## Features
+
+ - Channels
+   - Create new (group or personal)
+   - Edit group chat (change name and logo)
+   - Close channel
+   - Invite members
+   - Leave channel
+   - Pin/unpin channel in list
+   - Mute/unmute channel
+   - Last read message
+ - Messages
+   - Add/Edit/Delete
+   - Add message attachements
+   - Search messages
+   - Paginate messages
+ - Members
+   - Activate user
+   - Search members
+   - Subscribe/Unsubscribe on/from push notifications
+   - Change notification settings
+
+## Table of Contents
+
+1. [Core dependencies](#Core-dependencies)
+2. [Softeq dependent packages](#Softeq-dependent-packages)
+3. [Dependent projects](#Dependent-projects)
+4. [Additional documentation](#Additional-documentation)
+5. [Development setup](#Development-setup)
+6. [Comprehensive examples](#Comprehensive-examples)
+7. [About](#About)
+8. [Contributing](#Contributing)
+9. [License](#License)
 
-Product's dependencies are installed as NuGet-packages.
+## Core dependencies
 
-## Dependencies
+The following list contains all external core dependencies of NetKit.Chat projects:
 
-  - Microsoft.NETCore 2.0
+ - Microsoft.NETCore 2.0
```

Comment:
from my prospective it would be better to not show dependency versions. In that case we would need to not forget  to update all dependency when new version will be released

Reply:
Yes, but in this case we also may remove our internal packages paragraph because we also need to support that section.

Reply:
Let's do it